Buttons.h

```diff
@@ -0,0 +1,75 @@
+/**
+ * @file Buttons.h
+ * @author Kailen Swensen
+ * @brief Class to house Button class, modified for our purposes
+ * @version 0.1
+ * @date 2022-05-19
+ * 
+ */
+
+#include "Button.h"
+
+// basic print function for default callback functions
+void button_pressed()
+{
+  Serial.println("Button Pressed");
+}
+
+class Buttons
+{
+  public:
+
+  // defaulted to demo circuit values
+  Buttons() : gap(500), pin1(10), pin2(11), pin3(12), pin4(13)
+  {
+    setup();

```

Comment:
```suggestion

```

Reply:
Ah yes, that makes sense. I'll resolve with your changes. Thanks for catching that!